package.json

```diff
@@ -23,7 +23,7 @@
   "scripts": {
     "build": "npm run build:icon-index && npm run build:babel && npm run build:copy-files",
     "build:icon-index": "babel-node ./scripts/icon-index-generator.js",
-    "build:babel": "babel ./src --ignore *.spec.js --out-dir ./build",
+    "build:babel": "node --stack-size=10000 ./node_modules/.bin/babel ./src --ignore *.spec.js --out-dir ./build",
```

Comment:
Partly, at least it now processes svg-icons (on Windows) but in the end it errors out with:

```
npm ERR! material-ui-build@0.16.0-alpha build:babel: `node --stack-size=10000 ./node_modules/babel-cli/bin/babel.js ./src --ignore *.spec.js --out-dir ./build`
npm ERR! Exit status 3221225725
npm ERR!
npm ERR! Failed at the material-ui-build@0.16.0-alpha build:babel script 'node --stack-size=10000 ./node_modules/babel-cli/bin/babel.js ./src --ignore *.spec.js --out-dir ./build'.
```

Not sure yet what it means :)